Jenkinsfile

```diff
@@ -9,7 +9,7 @@ pipeline {
   }
 
   libraries {
-    lib("pay-jenkins-library@master")
+    lib("pay-jenkins-library@PP-2497_decrease_frontend_image_size")
```

Comment:
So for backwards compatability, this will need to remain the `PP-2497_decrease_frontend_image_size`. Once that branch has been merged we should switch it over to `master`.